iothub/service/src/DirectMethod/Models/DirectMethodClientResponse.cs

```diff
@@ -18,7 +19,8 @@ public class DirectMethodClientResponse
         /// To unit test methods that use this type as a response, inherit from this class and give it a constructor
         /// that can set the properties you want.
         /// </remarks>
-        protected internal DirectMethodClientResponse()
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public DirectMethodClientResponse()
```

Comment:
This doesn't have a parameterized constructor defined, so we should be able to exclude this.

Reply:
We had it there for unit testing, but wanted to hide it because normally a user shouldn't be creating this object. I think in this case declaring the ctor but hiding it is the right thing to do.